conans/test/functional/command/profile_test.py

```diff
@@ -69,6 +74,10 @@ def show_test(self):
     def profile_update_and_get_test(self):
         client = TestClient()
         client.run("profile new ./MyProfile --detect")
+        if "WARNING: GCC OLD ABI COMPATIBILITY" in client.out:
+            self.assertIn("edit the MyProfile profile at", client.out)
+            self.assertIn("profiles/MyProfile", client.out)
```

Comment:
This is still wrong, because the profile is not under the ``profiles`` subfolder, but in a completely different folder (the current one of the user cwd)

Reply:
So if the user pass a path instead of just the profile name, we should print:

```
If you want Conan to use the new ABI, edit the MyProfile profile at:

    /home/custom/dir/profiles/MyProfile
```

Is it correct?

Reply:
Not sure where that path comes from. In the code I still see ``~/.conan/profiles/{profile}`` so it would be very difficult that it appears the real path to the file, which will be ``os.path.join(client.current_folder, "MyProfile")``

Reply:
`os.path.join(client.current_folder, "MyProfile")` solves this specific case, but the user should be able to pass an absolute path too, which means the profile file path showed should be the same.

There are two lines, the first one has the profile name. So independent which path the user passed, it only should show the file name (profile name).

The second line is about the profile file path. The user is able to pass any path, including relative or absolute, or even the profile name only, thus:
- If the profile name is ONLY a name, then we should append the profile folder with the profile name. However, we are using a hardcoded folder. I would prefer to use `PROFILES_FOLDER` from `conans.client.cache.cache` in this case.
- If the profile name is a RELATIVE PATH, then we should transform it in absolute file path and show on the second line.
- If the profile name is an ABSOLUTE PATH, then just forward the path on the second line (do nothing).

WDYT ?

Reply:
@uilianries yes, I think that should be the best fix

Reply:
I made new changes based on comments above. Please, review again :eyeglasses: